.travis.yml

```diff
@@ -21,3 +38,12 @@ addons:
       - xvfb
       - g++-4.8
       - libgnome-keyring-dev
+      - createrepo
+
+env:
+  global:
+    - secure: "SnC/JYCTk4sr6pI5ae8nB7MZrmckjVqTvwk0fCMIRzzbSRSLHnYyOsfMXalGy1dKK7k7sL5INlo8iSEqQAi4ScRSlE/0uPIa822sAFIp35kiWuGJzOWcplfBc9j6ecS6TsgHF15EAUgGZ89r932Swk2RK4nCL5HCVmrxc5moCcraGGIy80STkDdJaZbHjVGl9KwnOTW8iHbxo/Emd3r/yFH6MdFLWLYfgNiT36dzGZIl+sfQomjWyx3HhepsBiFXQYJgDpuiY8reOvRsbDwZUdbxDxbBQKwL3FMDOHgAzDArlY+8Dja7CBKyj+G5SFa8rN3RUNVuC2UyC2ZCb5tdolyxVB+zywrVnJYXAa6icdL7hMejrkn+lVht0tnx0NXjzFL/AI+hfOCm2CvQYi/Fezr2rXgQh2H35RaRFaft9tZsLv32KfJCwSIIDwnRZ8YCqb2cYqXp+pqR6smCd+i+USvV7FvaFdygyTDghnctDA/h67PVW1ZaUVV3In3L3fuI4ON7JUdhnyZVVsRIy30dHLw9WhUNFQqld0bu1MM7Q3QDZRAsgCB4JrulKtn88rXlA+3Zw0kZzOvUpJ7V3GaN8bE8lru6ZY6V8MWqX3oid0dO2HE4Z69Ev5L/N0E0Kh8NS9aA4Vu0mmiuDn6TPIRtOsV1hNmvzJB1PhTuBs4iUSo="
```

Comment:
After this is merged let's confirm this is legit. See https://github.com/brave/internal/issues/14